### PR TITLE
Feature add example for f3 board

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: application/mcb-master-coc.c [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+    /** Set mapping and move MCB to cyclic state.
+     * Return number can be:
+     *    -> errorcode if failed
+     *    -> cyclic size if successful */
+    int16_t i16CycSt = SetMcb0CyclicMode();
+
+    if (i16CycSt > NO_ERROR)
+    {
+    	/** Cyclic state has been reached successfully */
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-coc.c]: wrong tab

File: application/mcb-master-coc.c [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+{
+    int32_t i32Ret = NO_ERROR;
+
+    /** The config message is managed by the cyclic process function,
+    *   a single config message needs a set of multiple cyclic process to
+    *   receive the config message answer. */
+    if ((eCoCResult == MCB_READ_SUCCESS) || (eCoCResult == MCB_STANDBY))
+    {
+    	/** Clear the message and send a new request */
+    	tMcbMsg.eStatus = MCB_STANDBY;
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-coc.c]: same wrong iden

File: application/mcb-master-coc.c [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+void AppCyclicProcess(void)
+{
+    if (ptMcbInst[MCB_INST0].isCyclic != false)
+    {
+        /** Perform a cyclic transfer */
+        Mcb_CyclicProcess(&(ptMcbInst[MCB_INST0]), &eCoCResult);
+        u32CycCnt++;
+
+        /** Copy VBus variable to local */
+        memcpy((void*)&u32VBusRead, (const void*)ppTxDatPoint[1], sizeof(u32VBusRead));
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-coc.c:149]: set a define VBUS_MAPPING_INDEX

File: application/mcb-master-cyclic.c [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+/** Error codes */
+#define NO_ERROR                    (int16_t)0
+#define MCB_CURRENT_STATUS_ERROR    (int16_t)-1
+#define MCB_MAPPING_ERROR           (int16_t)-2
+
+#define MCB_TX_MAP_NMB              (uint16_t)2U
+#define MCB_RX_MAP_NMB              (uint16_t)2U
+
+int16_t
+SetMcb0CyclicMode(void);
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-cyclic.c:35]: doc

File: application/mcb-master-cyclic.c [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+void AppStart(void)
+{
+    /** Set mapping and move MCB to cyclic state */
+    int16_t i16CycSt = SetMcb0CyclicMode();
+
+    if (i16CycSt > NO_ERROR)
+    {
+        /** Set a new current Q setpoint */
+        float fCurrentQSP = (float)1.1f;
+        memcpy(ppRxDatPoint[1], (const void*)&fCurrentQSP, sizeof(float));
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-cyclic.c:80]: same, use mapping index to help to understand the structure usage

**Review by jcastro-ingeniamc — CHANGES_REQUESTED** [review]

File: application/mcb-master-coc.c (hunk 1 above)
Comment (jperez-ingenia) [reply application/mcb-master-coc.c]: fixed

File: application/mcb-master-coc.c (hunk 2 above)
Comment (jperez-ingenia) [reply application/mcb-master-coc.c]: fixed

File: application/mcb-master-coc.c (hunk 3 above)
Comment (jperez-ingenia) [reply application/mcb-master-coc.c:149]: I'll do it now in my branch... I'll do more improvements

File: application/mcb-master-cyclic.c (hunk 4 above)
Comment (jperez-ingenia) [reply application/mcb-master-cyclic.c:35]: done

**Review by jperez-ingenia — APPROVED** [review]

**Pull request merged**